tests/Js/Main/MiscTests.fs

```diff
@@ -468,7 +468,7 @@ type Order =
         quantity : int<kg>
     }
 
-#if !FABLE_COMPILER_TYPESCRIPT
+#if FABLE_COMPILER && !FABLE_COMPILER_TYPESCRIPT && !NPM_PACKAGE_FABLE_COMPILER_JAVASCRIPT
```

Comment:
@MangelMaxime I wonder if we can simplify this to be just:
```fs
#if !FABLE_COMPILER
```
instead of introducing yet another define:
```fs
#if FABLE_COMPILER && !FABLE_COMPILER_TYPESCRIPT && !NPM_PACKAGE_FABLE_COMPILER_JAVASCRIPT
```